$NetBSD$

Add upstream change to fix build failure with latest glibmm.

--- src/gselector.cpp.orig	2009-12-07 21:43:35.000000000 +0000
+++ src/gselector.cpp
@@ -16,6 +16,8 @@
  * Software Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
  */
 
+#include <glibmm/main.h>
+
 #include "gselector.hpp"
 
 namespace
